package.json

```diff
@@ -41,6 +42,7 @@
     "chai": "^4.2.0",
     "eslint": "^8.0.1",
     "mocha": "^10.0.0",
-    "rollup": "^2.38.5"
+    "rollup": "^2.38.5",
+    "tinybench": "^2.5.0"
```

Comment:
Sorry for the delay. This looks great! Can you avoid the extra dependency? Perhaps using https://nodejs.org/api/perf_hooks.html 🤔

Reply:
Thank you for reviewing the pull request. I understand your concerns about adding tinybench. I chose it for its ease of use and detailed output. Over time, I've developed parsers tailored for its format in zka.lc, which allows for a deeper analysis of results.

While Node.js' perf hooks are useful, tinybench provides certain advantages in terms of comprehensiveness and user-friendliness. Transitioning now would require a significant overhaul of both benchmarks and the associated toolchain.

I'm open to discussing a potential compromise that might benefit us both.